src/core/Akka/Dispatch/SysMsg/ISystemMessage.cs

```diff
@@ -27,7 +27,7 @@ public interface ISystemMessage : INoSerializationVerificationNeeded
     /// <summary>
     ///     Class NoMessage.
     /// </summary>
-    public sealed class NoMessage : ISystemMessage
+    /*TODO: this class is not used*/public sealed class NoMessage : ISystemMessage
```

Comment:
We do not use this one yet, but it is used in the mailbox impl in scala. We need to look into its purpose